docs/volume-management.md

```diff
@@ -172,3 +172,18 @@ Suspended volumes can be resumed once they are fixed. Upon resuming, the corresp
 ```sh
 > kubectl directpv resume volumes --nodes node-1 --drives dm-3
 ```
+
+## SELinux in OpenShift:
+
+If you encounter the `relabel failed` error after executing the `suspend` or `resume` commands, you should set `spc_t` at the Tenant level Specification, as demonstrated below:
```

Comment:
1. Do we have version information of openshift like > v4.12?
2. Pods using suspended drives/volumes fail due to kubelet try to do selinux relabeling because of `lsetxattr` syscall fails on read-only filesystem. This issue would occur on any pod consuming suspended volume not limiting to MinIO Tenant pods. We would need to cover this information as well.